dashboard/app/controllers/api/v1/pd/workshop_enrollments_controller.rb

```diff
@@ -58,9 +58,14 @@ def create
     else
       ActiveRecord::Base.transaction do
         enrollment = ::Pd::Enrollment.new workshop: @workshop
-        enrollment.update!(enrollment_params.merge(school_info_attributes: school_info_params))
 
-        user&.update_school_info(enrollment.school_info)
+        if @workshop.course == 'Build Your Own Workshop'
```

Comment:
nit: is it possible to use the constant here as well?

Reply:
Oh yep, just found it's actually in the shared constants my bad! I'll get right on that!